doc/decision-tree-widget/tree.yml

```diff
@@ -287,6 +287,7 @@ children:
       - label: I want to re-subscribe
         children:
         - label: repeat
+        - label: repeatWhen
```

Comment:
To clarify how this file works, you have to add a description label before adding `children:` > `- label: repeatWhen` as the child of the description label. Just check how all other operators are formatted in this file. Not `repeatWhen` right under `repeat`.